hs_core/search_indexes.py

```diff
@@ -35,31 +40,63 @@ def normalize_name(name):
 
     However, the actual name parser in `probablepeople` is unnecessarily complex,
     so that strings that it determines to be human names are parsed instead by
-    `nameparser`.
+    the simpler `nameparser`.
 
     """
-    sname = name.encode('utf-8').strip()  # remove spaces
+    sname = name.strip()  # remove leading and trailing spaces
```

Comment:
I don't understand the implication of not performing the utf-8 casting

Reply:
utf-8 encode was added in #3324 to avoid a UnicodeEncodeError.  @alvacouch may have removed it because of the new regex?  I'm not sure.

Reply:
The reason that I changed this is that 

1. The name parser works in unicode. Casting to utf-8 actually breaks it for foreign names. 
2. I modified the default output format for printing and logging to utf-8 for all of HydroShare. This solves a world of printing problems.